deploy/roles/citus_coordinator_standby/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Move the data dir
+  become: True
+  become_user: "{{ pp_superaccount }}"
+  command: "mv /var/lib/pgsql/10/data /var/lib/pgsql/10/data.bk"
```

Comment:
@rioh Should the standby's postgresql service be stopped before moving the data folder?

Reply:
yup!  I had it, but must have deleted it when I was cleaning up the branch.  Thanks!